res/effects/chains/Echoverb HP.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version='1.0' encoding='utf-8'?>
+<EffectChain>
+ <Name>Echoverb HP</Name>
+ <MixMode>DRY+WET</MixMode>
+ <SuperParameterValue>0</SuperParameterValue>
+ <Effects>
+  <Effect>
+   <MetaParameterValue>0</MetaParameterValue>
+   <Id>org.mixxx.effects.threebandbiquadeq</Id>
+   <BackendType>BuiltIn</BackendType>
+   <Parameters>
+    <Parameter>
+     <Id>low</Id>
+     <Value>0.34375</Value>
+     <LinkType>NONE</LinkType>
+     <LinkInversion>0</LinkInversion>
+     <Hidden>0</Hidden>
```

Comment:
What about:

```suggestion
     <LinkType />
     <LinkInversion>false</LinkInversion>
     <Hidden>false</Hidden>
```

Reply:
That is not how it was in 2.1, so I don't think it should be changed.

Reply:
OK